src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -0,0 +1,103 @@
+package codeu.controller;
+
+import codeu.model.data.Conversation;
+import codeu.model.data.Message;
+import codeu.model.data.User;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
+import codeu.model.store.basic.UserStore;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+
+public class ProfileServlet extends HttpServlet {
+
+    /** Store class that gives access to Users. */
```

Comment:
sorry for constant comments on this - I keep seeing either +4 indent or +2 indent, and I'm no longer sure which one I advised where. Let's go with +2 spaces for indent from here on out